utils/filter_users.ts

```diff
@@ -2,19 +2,22 @@
 // See LICENSE.txt for license information.
 import {UserSearchOptions, UserListOptions, UserFilters} from 'utils/constants';
 
+export type FilterOptions = {
```

Comment:
There's actually a TypeScript provided type you can use here instead of defining your own dictionary:
`Record<string, string | boolean>`